frontend/aml_dashboard/src/index.js

```diff
@@ -223,14 +223,18 @@ b_train_AML.$click.subscribe( async () => {
   })
   .then(response => response.json())
   .then(async json => {
-    console.log(json);
-    console.log('RECEIVED');
+    if (json.Error) {
```

Comment:
change 
`textAMLTrainStatus.$value.set('<h1>Training...</h1> <br> <img src="https://i.gifer.com/origin/05/05bd96100762b05b616fb2a6e5c223b4_w200.gif">');`
to 
`textAMLTrainStatus.$value.set('<h2>Training...</h2> <br> <img src="https://i.gifer.com/origin/05/05bd96100762b05b616fb2a6e5c223b4_w200.gif">');`